crates/rattler_virtual_packages/src/lib.rs

```diff
@@ -229,6 +261,29 @@ impl From<LibC> for VirtualPackage {
     }
 }
 
+impl EnvOverride for LibC {
+    fn default_env_name() -> String {
+        "CONDA_LIBC_OVERIDE".to_string()
```

Comment:
Conda only allows overriding the glibc family version it seems:

https://github.com/conda/conda/blob/77aa3d337963fdbe5c817304a2052ae0125fd326/docs/source/user-guide/tasks/manage-virtual.rst#overriding-detected-packages

Reply:
Does rattler need to adhere to that?